wikipron/config.py

```diff
@@ -120,13 +127,20 @@ def _get_casefold(self, casefold: bool) -> Callable[[Word], Word]:
         return str.casefold if casefold else lambda word: word  # noqa: E731
 
     def _get_process_pron(
-        self, no_stress: bool, no_syllable_boundaries: bool, no_segment: bool
+        self,
+        no_stress: bool,
+        no_syllable_boundaries: bool,
+        no_segment: bool,
+        no_tone: bool,
     ) -> Callable[[Pron], Pron]:
         processors = []
         if no_stress:
             processors.append(functools.partial(re.sub, r"[ˈˌ]", ""))
         if no_syllable_boundaries:
             processors.append(functools.partial(re.sub, r"\.", ""))
+        if no_tone:
+            processors.append(functools.partial(unicodedata.normalize, "NFD"))
```

Comment:
Why is the NFD conversion necessary here? 

So if this flag enabled our Korean data comes out one _jamo_ per codepoint, but if it's not it comes out one _hangul_ per codepoint. That seems suboptimal to me.

Reply:
The NFD conversion is to handle the accents that Jackson referred to such as those used [here](https://en.wiktionary.org/wiki/beseni#Chichewa). `_TONES_REGEX` targets several of these combining diacritics, though thus far the ‘acute’ accent is the only one I’ve found used in transcriptions. 
I was wondering whether it was appropriate to do this conversion and am not sure if it will have any undesirable side effects, but since `segments.Tokenizer()` does it I assumed it would be okay to do it here as well.

Reply:
I didn't know that about `segments` applying NFD.

Let me suggest that we apply NFD to all pronunciations, immediately, rather than piecemeal (if segmentation and/or no-tones are enabled) before the configurable processing begins, presumably not in the config but earlier.

NFD conversion is _idempotent_ (is that the right word? a second application does nothing, in other words), so a second application by `segments` is harmless, and fast.

I wonder if we also want to apply NFC at the very end, also, because I consider that to be the standard representation for Unicode data at rest, and the graphemes are all NFC because Wiktionary (and the vast majority of web text in general) is all NFC. 

Sorry to add mission creep to this but I think those two steps (NFD on input, NFC on output) would be wise, WYDT?